website/blog/2021-11-09-blog1.md

```diff
@@ -0,0 +1,48 @@
+
+Additionally, because most whole-genome sequencing aims for 30X coverage, most confidences are around 90 or higher. With that, we can represent large intervals of high coverage with a single entry, indicating all high-quality homRef genotypes.
+
+The reblocked GVCFs are sufficient for multiple downstream applications. While some applications, like population variation surveys, can establish variant frequency using binary (i.e. good/bad) homRef confidences, other applications, like de novo analysis, require more nuance. For the latter kind of applications where the reference status of the parents is critical, the reblocked GVCFs provide multiple intermediate categories for homRef confidences. 
+
+Overall, reblocked GVCFs reduce the storage and I/O required to operate on variant data. This leads to faster, more efficient, and less expensive call sets. It is standard practice at the Broad to reblock GVCFs before joint calling. Now, we are making that step a standard part of the GVCF generation in the single sample pipeline. 
+
+## How is reblocking implemented in the pipelines?
+We’ve added reblocking as a default subtask, called “Reblock”, in the WARP [VariantCalling WDL script](https://github.com/broadinstitute/warp/blob/develop/pipelines/broad/dna_seq/germline/variant_calling/VariantCalling.wdl) which is imported by both the Exome and Whole Genome workflows. The Reblock task performs reblocking by using the GATK tool [ReblockGVCF](https://gatk.broadinstitute.org/hc/en-us/articles/360037593171). 
+
+While the reblocking option is turned on by default for any pipeline using the VariantCalling task, you can optionally turn it off by setting the `skip_reblocking` input in the workflows input configuration file (JSON). For example, if you’re working with the Exome pipeline, just add the parameter:
+
+```WDL
+ExomeGermlineSingleSample.BamToGvcf.skip_reblocking": true
+```
+
+We recognize that some researchers might want to perform reblocking independent of the Exome or Whole Genome pipelines, so we also created a utility workflow to reblock GVCFs. This pipeline takes in GVCFs produced by the single sample pipelines and outputs equivalently reblocked GVCFs. It is available in both the [WARP repository](https://github.com/broadinstitute/warp/tree/develop/pipelines/broad/dna_seq/germline/joint_genotyping/reblocking) and in a Terra workspace, and costs less than $0.02 per genome to run.
+
+If you’re curious as to whether your WARP pipeline output is a reblocked GVCF, just look at the filename suffix; a reblocked GVCF will have the `.rb.g.vcf` ending. 
+
+As always, if you have any questions about working with the updated workflows, [file an issue in WARP](https://github.com/broadinstitute/warp/issues) or reach out to [Kylee Degatano](mailto:kdegatano@broadinstitute.org).
+
+## Additional resources
```

Comment:
now that the reblocking schema and history is below maybe it should be linked here?

Reply:
I didn't link because we removed the spec.